sites/footers/no-footers-en.md

```diff
@@ -0,0 +1,43 @@
+---
+altLangPage: no-footers-fr.html
+breadcrumbs:
+- title: Footer
+  link: sites/footers/footers-en.html
+dateModified: 2023-01-16
+includes:
+  footer: edge
+language: en
+noFooterCorporate: 'true'
+noFooterContextual: 'true'
+noFooterMain: 'true'
+secondlevel: false
+share: 'true'
+title: Sub-footer band only, with no optional links
+titleH1: Display sub-footer band only, with no optional links
+---
+<div class="wb-prettify all-pre hide"></div>
+
+{% include alert-softlaunch.html component="site footer" version="4" %}
+
+The global (site-wide) footer at the bottom of each web page contains up to 3 bands:
+* Contextual band
+* Main band
+* Sub-footer band
```

Comment:
Utilise soit la minuscule partout ou alors la majuscule partout. Pas un mix des deux svp.
```suggestion
* contextual band
* main band
* sub-footer band
```

Reply:
We will uppercase them all

Reply:
I made all of them uppercase for the English examples